config/settings/staging.yml

```diff
@@ -6,4 +6,5 @@ preassembly_bundle_directory: '/dor/staging/integration-tests/image-test'
 preassembly_host: 'sul-preassembly-stage'
 preassembly_url: 'https://sul-preassembly-stage.stanford.edu'
 sdrapi_url: 'https://sdr-api-stage.stanford.edu'
-purl_url: 'https://purl-stage.stanford.edu'
+dor_services:
```

Comment:
any reason to keep two levels here?   not a big deal, just curious.

Reply:
@ndushay It's arbitrary, admittedly, but I did this to be consistent with the pattern we use everywhere else:

```yaml
dor_services:
  url: 'foo'
  token: 'bar'
```